package.json

```diff
@@ -28,7 +28,7 @@
     "react-intl": "^2.8.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.0",
+    "@babel/core": "^7.17.8",
```

Comment:
this doesn't seem related (and earned you a special packaging review, which shouldn't be needed otherwise?)

Reply:
Thanks, fixed.